docs/best_practice_guide.md

```diff
@@ -0,0 +1,127 @@
+# Best practice guide
+
+## What to use for
+
+You can use typo3-rector in various ways:
+- checking existing code if there are left out segments of the last upgrade
+- reformating e.g. typoscript files based on your editorconfig
+- evaluate upgrades and see what parts of your custom setup will be updated automatically
+- partial execution of full core upgrades
+
+
+
+
+## Guide for a good upgrade
+
+
+### Starting
+Starting with an upgrade should start with installing typo3-rector and checking for the rector rules/sets of your current version, not the one you're upgrading to.
+Often there are things that were missed out in previous upgrades while rector adds rulesets for those.
+
+One thing you should definitely need for upgrades with two or more version steps is the class-alias-maps.
```

Comment:
This is not true anymore. I fixed it with stubs.

Reply:
so the whole class alias map segment is obsolete?

Reply:
Yes, it is. Finally. Have a look at: https://github.com/sabbelasichon/typo3-rector/tree/main/stubs
This is taken into account with Static Reflection.